challenges/primitive-data-types/tests.ts

```diff
@@ -1,5 +1,5 @@
 type test_age = Expect<NotLiteral<number, typeof age>>;
-type test_name = Expect<NotLiteral<string, typeof name>>;
+type test_name = Expect<NotLiteral<string, typeof artistName>>;
```

Comment:
if you're going to change the name then also change the name on the test

```suggestion
type test_artistName = Expect<NotLiteral<string, typeof artistName>>;
```